isovar/dataframe_builder.py

```diff
@@ -133,6 +146,9 @@ def add(self, variant, element):
                 name = self.rename_dict[name]
             self.columns_dict[name].append(value)
 
+        for column_name, fn in self.extra_column_fns.items():
+            self.columns_dict[column_name] = fn(variant, element)
```

Comment:
Would it simplify the code to make `"gene"` an `extra_column_fn`? A little weird to have `chr/pos/ref/alt` + `gene` (one of these things is not like the other)?